### ISSUE TITLE
Error Reacting Message

### ISSUE THREAD

**Issue by SolSecSociety** [V4NSH4J/discord-mass-DM-GO#444]
Hi all, i get an error while trying to react a message. 
The error i get from each token is:
403 Forbidden
400 Bad Request

the procedure is:
Joining the bot to the server,
then from console "4" + 1 + Channel ID + Message ID + Reaction (text format ex :computer: )
But not working, any thought ?

**Comment by V4NSH4J** [comment]
Hey, looks like you're adding reaction from a message. In this case, you'd be given a list of emojis and their count. You have to enter the number before them (their number in the list)

**Comment by SolSecSociety** [comment]
Sorry my bad, i'm doing: 4 + 2 + Channel ID + Message ID + :computer:   but nothing i get those errors: 
403 Forbidden
400 Bad Request

**Comment by V4NSH4J** [comment]
Have you ensured your tokens are present in the server?

**Comment by SolSecSociety** [comment]
Yes Ofc, first i join then, then i try to react on the verification message following the steps above. But get this error. if i want to try to react from message, how it works ?  what this means ? "
You have to enter the number before them (their number in the list)"
